ansible/roles/test/files/ptftests/vxlan-decap.py

```diff
@@ -217,19 +266,21 @@ def work_test(self):
         try:
             for test in self.tests:
                 print test['name']
-                res_v, out_v = self.Vxlan(test)
-                print "  Vxlan            = ", res_v
+
                 res_f, out_f = self.RegularLAGtoVLAN(test)
                 print "  RegularLAGtoVLAN = ", res_f
+                self.assertTrue(res_f, "RegularLAGtoVLAN test failed:\n  %s\n\ntest:\n%s" % (out_f, pformat(test)))
+
                 res_t, out_t = self.RegularVLANtoLAG(test)
                 print "  RegularVLANtoLAG = ", res_t
-                print
+                self.assertTrue(res_t, "RegularVLANtoLAG test failed:\n  %s\n\ntest:\n%s" % (out_t, pformat(test)))
+
+                res_v, out_v = self.Vxlan(test)
+                print "  Vxlan            = ", res_v
                 if self.vxlan_enabled:
                     self.assertTrue(res_v, "VxlanTest failed:\n  %s\n\ntest:\n%s"  % (out_v, pformat(test)))
                 else:
                     self.assertFalse(res_v, "VxlanTest: vxlan works, but it must have been disabled!\n\ntest:%s" % pformat(test))
-                self.assertTrue(res_f, "RegularLAGtoVLAN test failed:\n  %s\n\ntest:\n%s" % (out_f, pformat(test)))
-                self.assertTrue(res_t, "RegularVLANtoLAG test failed:\n  %s\n\ntest:\n%s" % (out_t, pformat(test)))
```

Comment:
Why two tests were removed?

Reply:
These asserts are moved to line 272 and line 276. The test results are checked immediately after running test.

Reply:
> Why two tests were removed?

These asserts are moved to line 272 and line 276. The test results are checked immediately after running test.